packages/open-next/src/overrides/proxyExternalRequest/fetch.ts

```diff
@@ -0,0 +1,28 @@
+import type { ProxyExternalRequest } from "types/overrides";
+import { emptyReadableStream } from "utils/stream";
+
+const fetchProxy: ProxyExternalRequest = {
+  name: "fetch-proxy",
+  // @ts-ignore
+  proxy: async (internalEvent) => {
+    const { url, headers, method, body } = internalEvent;
+    const response = await fetch(url, {
```

Comment:
do we need to try/catch?

Reply:
I don't think we want to do it here. If something goes wrong, we want to return the user 500 route, and it would require changing the type (or fetching the 500 directly) and doing it for every proxy
There is one try/catch in `requestHandler`, and we should add one in `middleware.ts` as well